common/src/autogluon/common/utils/utils.py

```diff
@@ -7,8 +7,16 @@
 import sys
 from typing import Dict, Any
 
+from ..version import __version__
```

Comment:
This might create import issues with IDEs since the file is auto-generated. Should we do try-except with this import too?

Reply:
We already input `__version__` in the root init file of each module, so it won't make a difference.

Ex: `common/__init__.py`:

```python3
from autogluon.common.utils.log_utils import _add_stream_handler 

from .dataset import TabularDataset
from .space import Space, Categorical, Real, Int, Bool
from . import metrics
from . import constants


_add_stream_handler()

from .version import __version__
```